pkg/guardian/redis_conf_store_test.go

```diff
@@ -136,22 +182,22 @@ func TestConfStoreFetchesSets(t *testing.T) {
 		t.Fatalf("got error: %v", err)
 	}
 
-	gotLimit, err := c.FetchLimit()
+	gotGlobalRateLimit, err := c.FetchGlobalRateLimitConfig()
 	if err != nil {
 		t.Fatalf("got error: %v", err)
 	}
 
-	gotReportOnly, err := c.FetchReportOnly()
+	gotGlobalSettings, err := c.FetchGlobalSettingsConfig()
 	if err != nil {
 		t.Fatalf("got error: %v", err)
 	}
 
-	gotRouteRateLimits, err := c.FetchRouteRateLimits()
+	gotRateLimits := c.FetchRateLimitConfigs()
 	if err != nil {
 		t.Fatalf("got error: %v", err)
 	}
 
-	gotJails, err := c.FetchJails()
+	gotJails := c.FetchJailConfigs()
 	if err != nil {
```

Comment:
since some of these functions no longer return errors, we can probably get rid of some of the error checks in these tests.

Reply:
Fixed by 5d4ddd6.